### ISSUE TITLE
Cannot tab backwards through a snippet choice

### ISSUE THREAD

**Issue by juanitogan** [microsoft/vscode#178336]
Type: <b>Bug</b>

Here's a test snippet that shows the problem:

```
	"Print to console": {
		"prefix": "log",
		"body": [
			"console.log('$1');",
			"$2",
			"${3|one,two,three|}",
			"$4"
		],
		"description": "Log output to console"
	},
```

You can still click out of order to edit but, even then, the tab order doesn't recover.

For a stronger test, put stop 3 between two stop 2s (not that it currently matters).

VS Code version: Code 1.76.2 (ee2b180d582a7f601fa6ecfdad8d9fd269ab1884, 2023-03-14T17:55:54.936Z)
OS version: Windows_NT x64 10.0.19045
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz (12 x 2592)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.77GB (4.51GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
wikitext|Row|3.7.0

</details>

**Comment by jrieken** [comment]
I assume by "tab backwards" you mean `Shift+Tab`, right? Please enable keybindings debug mode (F1 > Toggle Keyboard Shortcuts Troubleshooting) and check what command gets executed

**Comment by juanitogan** [comment]
What?  The standard procedure here is to close an issue before testing example code that someone goes to the trouble of narrowing down and reporting???

All I saw before that was a comment that looked like it was from a troll.

Well, whatever.  Since no one here cared to try my test snippet before closing the issue, here's a peek under the hood in case any curious persons come along: It's running `acceptAlternativeSelectedSuggestion` instead of `jumpToPrevSnippetPlaceholder` most of the time when tab reversing through a snippet choice (not all the time, weirdly, due to other factors like when interrupting the navigation with some other action).  Really, though, not a single surprise there given what I already reported, and not nearly as useful as the test snippet I already gave.

![image](https://user-images.githubusercontent.com/2923222/229899495-157d8fc9-de56-41be-9bfc-fb99f4f7b70d.png)
